src/Errata/LabelExtensions.cs

```diff
@@ -58,4 +58,26 @@ public static Label WithPriority(this Label label, int priority)
         label.Priority = priority;
         return label;
     }
+
+    /// <summary>
+    /// Sets the number of additional context lines to display above this label.
+    /// </summary>
+    /// <param name="label">The label.</param>
+    /// <param name="lines">The number of lines above the label to display as context.</param>
+    /// <returns>The same instance so that multiple calls can be chained.</returns>
+    public static Label WithContext(this Label label, int lines)
```

Comment:
Perhaps another name for this extension method such as `WithContextLines` or similar? What do you think?